README.md

```diff
@@ -56,6 +56,10 @@ compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:aztec:develop-SNAP
 
 When Aztec is ready, we'll publish the artifact in Maven.
 
+## Using ktlint
+
+The FluxC project uses [ktlint](https://github.com/shyiko/ktlint) for Kotlin linting. You can run ktlint using `./gradlew ktlint`, and you can also run `./gradlew ktlintFormat` for auto-formatting. There is no IDEA plugin (like Checkstyle's) at this time.
```

Comment:
FluxC ? What is FluxC 😁

Reply:
Argh, I can't even copy/paste text correctly ;(